packages/client/components/TeamPrompt/TeamPromptLastUpdatedTime.tsx

```diff
@@ -20,7 +20,19 @@ interface Props {
   updatedAt: string | Date
 }
 
+const UPDATE_INTERVAL_MS = 1000
+
 export default function TeamPromptLastUpdatedTime({updatedAt, createdAt}: Props) {
+  const [relativeCreatedAt, setRelativeCreatedAt] = useState(() => relativeDate(createdAt))
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      setRelativeCreatedAt(relativeDate(createdAt))
+    }, UPDATE_INTERVAL_MS)
+
+    return () => clearInterval(intervalId)
+  }, [createdAt])
```

Comment:
+1 Would it make sense to use `useRefreshInterval` here?  That way instead of manually updating the `relativeCreatedAt` on the interval, we just let the full component refresh itself, including the relative timestamp.  That should reduce this to a two-ish line change (just the constant and calling the hook).

Reply:
I'm not sure what's more readable in this case so I'll with the solution you mentioned as it's shorter, thanks!